src/pattern.c

```diff
@@ -415,8 +415,14 @@ pat2str(pattern_t p)
 void
 seq2str(const sequence_t *sp, char *buf, int len)
 {
+    int i;
+    int passes = sp->len;
+
+    for (i = 0; i < sp->len; i++)
+        if (sp->pat[i].ptype == PAT_VERIFY)
+            passes++;
```

Comment:
Did you mean for that to be `passes--`?  I'm assuming the goal is to _not_ count the verification pass (which makes sense to me).

Reply:
No - I'm trying to have it match the man page and other documentation, and the verify pass effectively adds two passes (I assume it's counted as two, given that dod is considered 4 passes) as it writes and then reads the value back. But the data structure used counts the number of patterns in it, so PAT_VERIFY counts as 1 pass right now in the scrub help output.

For example, the man page for dod has:

`       dod    4-pass  DoD  5220.22-M  section 8-306 procedure (d) for sanitizing removable and non-removable rigid disks which requires overwriting all addressable locations with a
              character, its complement, a random character, then verify.  NOTE: scrub performs the random pass first to make verification easier: random, 0x00, 0xff, verify.
`

Current scrub help output for dod has:

`  dod           3-pass   DoD 5220.22-M
`

With this change, the scrub help output for dod matches the man page:

`   dod           4-pass   DoD 5220.22-M

`

Reply:
Oh I see!  Thanks for explaining.  OK, sounds good.